src/pages/mesh/security.md

```diff
@@ -12,16 +12,16 @@ keywords:
 
 # Security
 
-The data plane used by API Mesh is public but requires a MeshID and an API Key to access. We recommend storing your API Key in a header, instead of using the API Key appended to your mesh's URL.
+The data plane used by API Mesh is public but requires a MeshID and an Adobe I/O account to access.
 
-Basic authorization is provided by your Mesh ID and API Key. Additionally, API Mesh honors any downstream authorization headers provided by your [sources](./basic/handlers/index.md).
+Basic authorization is provided by your Mesh ID and Adobe I/O. Additionally, API Mesh honors any downstream authorization headers provided by your [sources](./basic/handlers/index.md).
```

Comment:
@prabhuram93 is this still accurate?

Reply:
We need to remove this "Basic authorization is provided by your Mesh ID and Adobe I/O. "
Question to Mike, can we add any other security benefits like DDOS, WAF?

Reply:
@mcmains7812 we have a section on DDOS: https://developer.adobe.com/graphql-mesh-gateway/mesh/security/#ddos-and-rate-limiting

What should we say about the WAF?